tests/platform_tests/api/test_psu.py

```diff
@@ -45,6 +54,25 @@ def setup(self, platform_api_conn):
             except:
                 pytest.fail("num_psus is not an integer")
 
+    #
+    # Helper functions
+    #
+
+    def compare_value_with_platform_facts(self, key, value, psu_idx):
+        expected_value = None
+
+        if self.chassis_facts:
+            expected_psus = self.chassis_facts.get("psus")
+            if expected_psus:
+                expected_value = expected_psus[psu_idx].get(key)
+
+        if not expected_value:
+            logger.warning("Unable to get expected value for '{}' from platform.json file for PSU {}".format(key, psu_idx))
+            return
```

Comment:
Remove warning message and make the check mandatory. See SFP portion of https://github.com/Azure/sonic-mgmt/pull/2268 for examples.